server/app/com/xsn/explorer/models/persisted/Transaction.scala

```diff
@@ -88,24 +88,17 @@ object Transaction {
       size = tx.size
     )
 
-    (HasIO(transaction, inputs, outputs), getContract(tx))
+    (HasIO(transaction, inputs, outputs), isTposContract(tx))
   }
 
   /**
    * A transaction can have at most one contract
    */
-  private def getContract(tx: rpc.Transaction[rpc.TransactionVIN.HasValues]): Option[TPoSContract] = {
+  private def isTposContract(tx: rpc.Transaction[rpc.TransactionVIN.HasValues]): Boolean = {
```

Comment:
I'd rename this to `seemsTPoSContract`, because finding the format doesn't guarantee a contract